ENGAGEHF/Medications/ColorKeyRow.swift

```diff
@@ -0,0 +1,25 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct ColorKeyRow: View {
+    let color: Color
+    let description: String
```

Comment:
Since this would potentially collide with `CustomStringConvertible`, I would propose to use `title` or `text` instead - what do you think?